src/plots/plots.js

```diff
@@ -2561,9 +2551,10 @@ plots.doCalcdata = function(gd, traces) {
     for(i = 0; i < fullData.length; i++) calci(i, true);
     for(i = 0; i < fullData.length; i++) calci(i, false);
 
-    for(i = 0; i < calcInteractionsFuncs.length; i++) calcInteractionsFuncs[i](gd, calcdata);
+    plots.doCrossTraceCalc(gd);
```

Comment:
Hmm, I guess we don't need to export `doCrossTraceCalc` anymore since it's only used here...

Reply:
stopped exporting -> https://github.com/plotly/plotly.js/pull/2870/commits/0bf82c862047304d4c1d1c6f205f773b6ceec044